### PR TITLE
KS [SK832] Translations for web hosting 1_Click_module_managment 

### PR DESCRIPTION
**Pull request by Kazarus4**
Updates :

- Dates of "last updates" for all guides
- translation from FR-FR guides to the other guides and check OVHcloud links
- Updates these syntaxes : 
-- OVHCLOUD => OVHcloud
-- WORDPRESS => WordPress
-- JOOMLA => Joomla!
-- DRUPAL => Drupal
-- PRESTASHOP => PrestaShop

### CODE REVIEW

File: pages/web/hosting/cms_manage_1_click_module/guide.de-de.md [hunk 1]
```diff
@@ -75,10 +75,34 @@ In beiden Fällen wird Ihnen eine E-Mail zum Zurücksetzen des Passworts zugesan
...
+
 Um das Passwort für den Zugang zum Verwaltungsinterface Ihrer Website zu ändern, gehen Sie im [OVHcloud Kundencenter](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.de/&ovhSubsidiary=de) zu `Web Cloud`{.action}. Klicken Sie auf `Hosting-Pakete`{.action} und auf das betreffende Hosting und dann auf den Tab `1-Klick-Module`{.action}.
 
 Klicken Sie dann auf den Button `...`{.action} rechts in der Zeile für Ihr Modul und dann auf `Passwort ändern`{.action}. Klicken Sie auf `Bestätigen`{.action}. Sie erhalten in einigen Minuten per E-Mail einen Link um Ihr Passwort zurückzusetzen.
 
+> [!primary]
+>
+> Wenn Sie Ihr Passwort für den Zugang zum Verwaltungsinterface Ihres CMS aus den oben genannten Gründen nicht über das OVHcloud Kundencenter ändern können, finden Sie die offizielle Dokumentation zu den verschiedenen CMS, die auf unseren Webhosting-Angeboten installiert werden:
+>
+> - WordPress : <https://wordpress.org/support/article/resetting-your-password/
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manage_1_click_module/guide.de-de.md]: Please fix links format (check on all versions)

File: pages/web/hosting/cms_manage_1_click_module/guide.en-asia.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Next, click on the `...`{.action} button to the right of the line related to you
 
 > [!primary]
 >
-> If your website does not display properly, please refer to the OVHcloud web hosting guides in the [Troubleshooting](https://docs.ovh.com/asia/en/hosting/) section.
+> If your website does not display properly, please refer to the *OVHcloud* web hosting guides in the [Troubleshooting](https://docs.ovh.com/asia/en/hosting/) section.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manage_1_click_module/guide.en-asia.md]: Please revert italic

File: pages/web/hosting/cms_manage_1_click_module/guide.en-asia.md [hunk 3]
```diff
@@ -89,10 +113,10 @@ Otherwise, go to the `Multisite`{.action} tab. Note the `Root folder` name: this
 
 Then log on to the [FTP space](https://docs.ovh.com/gb/en/hosting/log-in-to-storage-ftp-web-hosting/) of your web hosting. Open the `Root folder` found earlier in the `Multisite`{.action} tab and browse in to the configuration file of your website:
 
-- WordPress: **"wp-config.php"** (the database name appears as **"DB_NAME"**).
-- Joomla!: **"configuration.php"** (the database name appears under **"public $db"**).
-- Drupal: **"settings.php"** (Go to the **"sites"** folder then open the **"default"** directory. The database name appears under the name **"database"**).
-- PrestaShop: **"parameters.php"** (To find it, go to the **"app"** folder, then click on **"config"**. The name of your module’s database will appear as **"database_name"**).
+- *WordPress* : **"wp-config.php"** (the database name appears as **"DB_NAME"**).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manage_1_click_module/guide.en-asia.md]: Please revert italic

File: pages/web/hosting/cms_manage_1_click_module/guide.it-it.md [hunk 4]
```diff
@@ -75,10 +75,34 @@ In entrambi i casi, riceverai un'email di reimpostazione della password.
...
+> - Drupal : L'editor di questo software non offre, alla data, la documentazione per modificare la password di accesso all'interfaccia di amministrazione di Drupal. Ti invitiamo a contattare direttamente l'editor su questo argomento. Per maggiori informazioni, consulta la pagina ufficiale [drupal.org](https://www.drupal.org/){.external}.
+> - PrestaShop : L'editor di questo software non offre, alla data, la documentazione necessaria per modificare la password di accesso all'interfaccia di amministrazione di PrestaShop. Ti invitiamo a contattare direttamente l'editor su questo argomento. Per maggiori informazioni, clicca [qui](https://www.prestashop.com){.external} per accedere alla pagina ufficiale.
+>
+È inoltre possibile modificare la password di accesso all'interfaccia di gestione del tuo CMS direttamente dal tuo database.<br>
+Tuttavia, in caso di difficoltà o dubbi, ti consigliamo di eseguire l'operazione utilizzando la documentazione proposta dal tuo CMS o di rivolgerti a uno specialista del settore (https://partner.ovhcloud.com/it/). OVH non potrà fornirti alcuna assistenza. Per maggiori informazioni consulta la sezione [Per saperne di più](#go-further) di questa guida.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manage_1_click_module/guide.it-it.md]: OVH > OVHcloud

File: pages/web/hosting/cms_manage_1_click_module/guide.pt-pt.md [hunk 5]
```diff
@@ -75,10 +75,34 @@ Em ambos os casos, ser-lhe-á enviado um e-mail de reinicialização da password
...
 
 A seguir, clique no botão `...`{.action} à direita da linha relativa ao seu módulo e depois `Alterar palavra-passe`{.action}. Clique em `Validar`{.action}. Receberá dentro de alguns minutos por e-mail um link de reinicialização da sua password.
 
+> [!primary]
+>
+> Se não pode alterar a palavra-passe de acesso à interface de gestão do seu CMS a partir da Área de Cliente OVHcloud pelas razões indicadas acima, poderá consultar a documentação oficial para os diferentes CMS propostos em instalação nos nossos alojamentos partilhados:
+>
+> - WordPress : <https://wordpress.org/support/article/resetting-your-password/>
+> - Joomla! : <https://docs.joomla.org/How_do_you_recover_or_reset_your_admin_password%3F>
+> - Drupal : O editor deste programa não propõe, à data, documentação para alterar a palavra-passe de acesso à interface de administração do Drupal. Sugerimos que contacte diretamente o editor sobre este assumpto. Para mais informações, consulte a página oficial [drupal.org](https://www.drupal.org/) {.external}.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manage_1_click_module/guide.pt-pt.md]: Please fix links formatting

File: pages/web/hosting/cms_manage_1_click_module/guide.pt-pt.md [hunk 6]
```diff
@@ -140,10 +164,12 @@ Não instale no seu site plugins ou templates que não foram recomendados pelas
...
 - [PrestaShop](https://www.prestashop.com/pt){.external}
 
-## Quer saber mais? <a name="gofurther"></a>
+## Quer saber mais? <a name="go-further"></a>
 
 [Resolver os erros mais frequentes associados aos módulos 1 clique](https://docs.ovh.com/pt/hosting/erros-frequentes-modulos-em-1-clique/).
 
 Para serviços especializados (referenciamento, desenvolvimento, etc.), contacte os [parceiros OVHcloud](https://partner.ovhcloud.com/pt/).
 
+Se pretender usufruir de uma assistência na utilização e na configuração das suas soluções OVHcloud, consulte as nossas diferentes ofertas de suporte (https://www.ovhcloud.com/pt/support-levels/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manage_1_click_module/guide.pt-pt.md]: Please fix link

File: pages/web/hosting/cms_manage_1_click_module/guide.it-it.md [hunk 7]
```diff
@@ -140,10 +164,12 @@ Non installate sul vostro sito plugin o template che non sono stati consigliati
...
 - [PrestaShop](https://www.prestashop.com/it){.external}
 
-## Spingiti oltre <a name="gofurther"></a>
+## Spingiti oltre <a name="go-further"></a>
 
 [Risolvere gli errori più frequenti associati ai moduli in 1 click](https://docs.ovh.com/it/hosting/errori-frequenti-moduli-in-1-click/).
 
 Per prestazioni specializzate (referenziamento, sviluppo, ecc...), contatta i [partner OVHcloud](https://partner.ovhcloud.com/it/).
 
+Per usufruire di un supporto per l'utilizzo e la configurazione delle soluzioni OVHcloud, è possibile consultare le nostre soluzioni (offerte di supporto)(https://www.ovhcloud.com/it/support-levels/).
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/cms_manage_1_click_module/guide.it-it.md]: Please fix links

File: pages/web/hosting/cms_manage_1_click_module/guide.pt-pt.md (hunk 6 above)
Comment (Kazarus4) [reply pages/web/hosting/cms_manage_1_click_module/guide.pt-pt.md]: Done

File: pages/web/hosting/cms_manage_1_click_module/guide.it-it.md (hunk 4 above)
Comment (Kazarus4) [reply pages/web/hosting/cms_manage_1_click_module/guide.it-it.md]: Done

File: pages/web/hosting/cms_manage_1_click_module/guide.en-asia.md (hunk 2 above)
Comment (Kazarus4) [reply pages/web/hosting/cms_manage_1_click_module/guide.en-asia.md]: Done

File: pages/web/hosting/cms_manage_1_click_module/guide.de-de.md (hunk 1 above)
Comment (Kazarus4) [reply pages/web/hosting/cms_manage_1_click_module/guide.de-de.md]: Done

File: pages/web/hosting/cms_manage_1_click_module/guide.en-asia.md (hunk 3 above)
Comment (Kazarus4) [reply pages/web/hosting/cms_manage_1_click_module/guide.en-asia.md]: Done

File: pages/web/hosting/cms_manage_1_click_module/guide.it-it.md (hunk 7 above)
Comment (Kazarus4) [reply pages/web/hosting/cms_manage_1_click_module/guide.it-it.md]: Done

**Pull request merged**